[#95] Add option to prevent serializers from being wrapped (#96)
pinojs/pino-http: README.md

* `customSuccessMessage`: set to a `function (res) => { /* returns message string */ }` This function will be invoked at each successful response, setting "msg" property to returned string. If not set, default value will be used.
* `customErrorMessage`: set to a `function (res, err) => { /* returns message  string */ }` This function will be invoked at each failed response, setting "msg" property to returned string. If not set, default value will be used.
* `customAttributeKeys`: allows the log object attributes added by `pino-http` to be given custom keys. Accepts an object of format `{ [original]: [override] }`. Attributes available for override are `req`, `res`, `err`, and `responseTime`. 

> * `wrapSerializers`: when false, custom serializers will be passed the raw value directly. Defaults to true

Reviewer: ```suggestion
* `wrapSerializers`: when `false`, custom serializers will be passed the raw value directly. Defaults to `true`.
```

    res: pino.stdSerializers.res
  },

>   // Prevent serializers from being wrapped with std serializers
>   // (useful if you want to honor the serializers defined by `opts.logger`)
>   wrapSerializers: false,

Reviewer: ```suggestion
  // Set to `false` to prevent standard serializers from being wrapped.
  wrapSerializers: true,
```

})
```

If you prefer to work with the raw value directly, you can pass in `opts.wrapSerializers` as `false`:

```js
var http = require('http')
var logger = require('pino-http')({
  wrapSerializers: false,
  serializers: {
    req (req) {
      // `req` is the raw `IncomingMessage` object, not the already serialized request from `pino.stdSerializers.req`.
      return {
        message: req.foo
      };
    }
  }
})
```

> 

Author: I wasn't quite sure of the best name for the option, some ideas were:

wrapSerializers: false
preSerialize: false
raw: true
serializeRaw: true

Reviewer 2: `wrapSerializers` is ok.
